lib/generators/install/templates/index.coffee.erb

```diff
@@ -0,0 +1,8 @@
+#= require cable
+#= require_self
+#= require_tree .
+
+@<%= application_name %> ?= {}

```

Comment:
Perhaps we can set aside some dedicated objects on `Cable` for storing the consumer and subscription instances instead of creating another (potentially conflicting) global.

Something like:
```coffee
Cable.consumer = Cable.createConsumer(...)
```
and
```coffee
Cable.subscriptions["<%= channel_name %>"] = Cable.consumer.subscriptions.create(...)
```